src/components/app/App.js

```diff
@@ -1,8 +1,13 @@
+import Header from "../header/header";
 import { AppWrapper } from "./styledApp";
+import { headerLinks } from '../../constants/headerConstants';
 
 function App() {
   return (
-    <AppWrapper></AppWrapper>
+    <AppWrapper>
+      <Header headerLinks={headerLinks} />
```

Comment:
i'm curious why you passed this as a prop vs importing in the header component file?